src/lib/Libnet/net_server.c

```diff
@@ -1035,6 +1019,25 @@ init_poll_context(void)
 		log_err(errno, __func__, "could not initialize poll_context");
 		return (-1);
 	}
+#ifdef WIN32
+	/* set a dummy fd in the read set so that	*/
+	/* select() does not return WSAEINVAL 		*/
+	sd_dummy = socket(AF_INET, SOCK_STREAM, 0);
+	if (sd_dummy < 0) {
+		errno = WSAGetLastError();
+		log_err(errno, __func__, "socket() failed");
+		return -1;
+	}
+	if((tpp_em_add_fd(poll_context, sd_dummy, EM_IN) == -1)){
```

Comment:
Minor nit - please add a space between ) and {

Reply:
Fixed